engine/src/main/resources/templates/Vulnerability/dailyEmailBody.html

```diff
@@ -0,0 +1,46 @@
+{#include Vulnerability/insightsEmailBody}
+{#content-title}Red Hat has published CVEs that require your attention - {action.timestamp.toUtcFormat()}{/content-title}
+{#content-body}
+<!-- Body section - Empty row to have more padding between content-->
+<tr>
+    <td class="rh-body rh-multi-column">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td>
+                </td>
+            </tr>
+            <!-- end content block -->
+            </tbody>
+        </table>
+    </td>
+</tr>
+<!-- end body section -->
+<!-- Body section -->
+<tr>
+    <td class="rh-body">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td class="rh-content__block">
+                    <p>
+                        Red Hat Insights has identified one or more CVEs that require your attention based on the notification setting configured. Please review the Insights Vulnerability service to determine risk and exposure for your organization.
+                        These are the identified CVEs.
+                        <ul>
+                            {#for cve in action.context.vulnerability.reported_cves}
+                            <li><a href="https://console.redhat.com/insights/vulnerability/cves/{cve}">{cve}</a></li>
```

Comment:
You could use the environment, so it links to stage when produced by the stage environment
```suggestion
                            <li><a href="{environment.url()}/insights/vulnerability/cves/{cve}">{cve}</a></li>
```

Reply:
Thanks, did not know about that one. I also updated the other templates to use the dynamic console link, in another commit.